napari/_vispy/utils/gl.py

```diff
@@ -119,6 +119,10 @@ def fix_data_dtype(data):
         cull_face=False,
         blend=True,
         blend_func=('src_alpha', 'one_minus_src_alpha', 'zero', 'one'),
+        blend_equation='func_add',
```

Comment:
Could you explain this change?

Reply:
It's the same bug as: https://github.com/vispy/vispy/pull/2324
To summarize: if a blending preset doesn't specify a `blend_equation` then it won't be set, meaning the last `blend_equation` will be used. 
This was fine when all the other options also used `func_add` but now the new mode `minimum` uses a different `blend_equation` (`min`) so toggling to `translucent_no_depth` from `minimum` would result in a black square. This was most apparent when using the default channel splitting options which use  `translucent_no_depth` as the base layer.